ui/00_message/skill/normal_skill_name_06.xml

```diff
@@ -2,17 +2,17 @@
   <message>
     <key>NORMAL_SKILL_NAME_06_92</key>
     <original>マジックボール</original>
-    <translation>Magic Balls</translation>
+    <translation>Magick Bolts</translation>
```

Comment:
For example, Devastate being in the warrior's skills despite the text of the skill not meaning Devastate.